app/adapters/column.js

```diff
@@ -15,6 +15,13 @@ export default DS.JSONAPIAdapter.extend({
     return this.get('settings.schemaNamespace');
   }),
 
+  ajaxOptions: function(url, type, options) {
```

Comment:
ES6 syntax is 
```
ajaxOptions() {
}
```
Since you don't use url, type or options, just leave them out. That's why the build is failing - checkstyle.

Reply:
They are used to call super function. I changed it.

Reply:
Just use ```this._super(...arguments)``` instead.

Example: https://github.com/yahoo/bullet-ui/blob/master/app/components/query-input.js#L42